README.md

```diff
@@ -59,7 +59,7 @@ Include using a script tag:
 Add the script tag to your HTML page, specifying the version you will use:
 
 ```html
-<script src="https://d10ka0m22z5ju5.cloudfront.net/js/scanthng/4.0.0/scanthng-4.0.0.js"></script>
+<script src="https://d10ka0m22z5ju5.cloudfront.net/js/scanthng/4.0.1/scanthng-4.0.1.js"></script>
```

Comment:
is this going to be published after merging this PR?

Reply:
Yes, `npm publish` followed by the Jenkins job that pushes the artifact to S3 to make this URL accurate.